OIPA/iati/parser.py

```diff
@@ -233,15 +233,10 @@ def add_activity(self, elem):
             default_currency = None
 
             hierarchy = self.return_first_exist(elem.xpath('@hierarchy'))
-            last_updated_datetime = self.return_first_exist(elem.xpath('@last-updated-datetime'))
-            # if last_updated_datetime:
-            #     last_updated_datetime = last_updated_datetime.split('+')[0]
-            #     last_updated_datetime = last_updated_datetime.split('-')[0]
-            # strp_time = time.strptime(last_updated_datetime_str, '%Y-%m-%d %H:%M:%S')
-            # last_updated_datetime = datetime.fromtimestamp(time.mktime(strp_time))
+            last_updated_datetime = self.return_first_exist(elem.xpath('@last-updated-datetime')) or ""
 
-            linked_data_uri = self.return_first_exist(elem.xpath('@linked-data-uri'))
-            iati_standard_version = self.return_first_exist(elem.xpath('@version'))
+            linked_data_uri = self.return_first_exist(elem.xpath('@linked-data-uri')) or ""
+            iati_standard_version = self.return_first_exist(elem.xpath('@version')) or ""
```

Comment:
Tried that, but then it also returns an empty string when the variable is later saved as Decimal etc.  Since it's a total black box where this all happens, the current solution seemed safer.